### PR TITLE
ground items plugin: add loot beams

### PR DESCRIPTION
**Pull request by Trevor159**
picture for a blog: https://i.imgur.com/Yb9VyJY.png

gif for a blog: https://i.imgur.com/8QqQVO8.gif

Requires !92

new gif with colors
https://i.imgur.com/GIhE9IZ.gif

### CODE REVIEW

File: runelite-api/src/main/java/net/runelite/api/Client.java [hunk 1]
```diff
@@ -1707,4 +1707,8 @@
 	 * Makes all widgets behave as if they are {@link WidgetConfig#WIDGET_USE_TARGET}
 	 */
 	void setAllWidgetsAreOpTargetable(boolean value);
+
+	void addLootBeam(WorldPoint worldPoint);
```
Comment (Nightfirecat) [inline runelite-api/src/main/java/net/runelite/api/Client.java]: These could use some doc explaining what they do.

**Comment by Nightfirecat** [comment]
Code LGTM otherwise.

**Comment by dekvall** [comment]
This fixes #9050

**Comment by hayryne** [comment]
Has this been forgotten? Would love to see this!

**Comment by Trevor159** [comment]
updated to use a new `RuneliteObject` api which is also included in this PR
it also uses DBs suggestion and has colors now

new gif https://i.imgur.com/GIhE9IZ.gif

**Pull request closed without merging**